zip-0225.rst

```diff
@@ -7,13 +7,427 @@
           Sean Bowe <sean@electriccoin.co>
           Kris Nuttycombe <kris@electriccoin.co>
           Ying Tong Lai <yingtong@electriccoin.co>
-  Status: Reserved
+  Status: Draft
   Category: Consensus
   Discussions-To: <https://github.com/zcash/zips/issues/440>
 
-This ZIP number is reserved. Prelininary technical content related to the version 5 format
-is currently in:
+Terminology
+===========
 
-* the `draft Orchard protocol specification <https://raw.githubusercontent.com/daira/zips/orchard-circuit/protocol/orchard.pdf#txnencodingandconsensus>`_;
-* the `Orchard Book <https://zcash.github.io/orchard/design.html>`_;
-* `ZIP 244 <zip-0244.rst>`_.
+The key words "MUST" and "MAY" in this document are to be interpreted as described in
+RFC 2119. [#RFC2119]_
+
+The term "network upgrade" in this document is to be interpreted as described in ZIP 200
+[#zip-0200]_.
+
+Abstract
+========
+
+This proposal defines an update to the Zcash peer-to-peer transaction format to include
+support for data elements required to support the Orchard protocol [#protocol_nu5]_. The
+new transaction format defines well-bounded regions of the serialized form for to serve
+each of the existing pools of funds, and adds and describes a new region containing
+Orchard-specific elements.
+
+This ZIP also depends upon and defines modifications to the computation of the values
+**TxId Digest**, **Signature Digest**, and **Authorizing Data Commitment** defined by ZIP
+244 [#zip-0244]_.
+
+Motivation
+==========
+
+The new Orchard shielded pool requires a serialized data elements that are distinct from
```

Comment:
Should be "...shielded pool requires serialized data..."

Reply:
```suggestion
The new Orchard shielded pool requires serialized data elements that are distinct from
```